metrics_v2/handler/collectors/literal_collector.py

```diff
@@ -0,0 +1,23 @@
+from google.protobuf import duration_pb2
+
+from handler.collectors import base
+
+class LiteralCollector(base.BaseCollector):
+  def read_metrics_and_assertions(self):
+    for metric_key, assertion in self._source.assertions.items():
+      try:
+        # TODO: implement lookups in nested fields
+        raw_value = getattr(self._event, metric_key)
+      except AttributeError:
+        logging.error('Attribute %s not found in event', metric_key)
```

Comment:
That's correct. This alert will get rolled up with other error-level messages, and this will only print if the user specified that a value should be present.